tmux-mem-cpu-load.plugin.tmux

```diff
@@ -19,6 +19,14 @@
 CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 pushd $CURRENT_DIR #Pushd to the directory where this plugin is located.
-cmake .
-make
+
+# Attempt to rebuild the plugin and log any errors in the tmux display window.
+output=$(cmake . 2>&1) || tmux run-shell "echo \"'cmake $CURRENT_DIR' failed.
+$output
+\"" && exit 1
```

Comment:
It might make more sense not to `exit` here, if it's possible that `cmake` can exit with a non-zero status but still be able to `make` the final binary?